frontend/src/metabase/query_builder/components/view/ViewHeader/components/DashboardBackButton/DashboardBackButton.tsx

```diff
@@ -36,21 +35,21 @@ export function DashboardBackButton({
   const label = t`Back to ${dashboard.name}`;
 
   return (
-    <Tooltip tooltip={label}>
-      <BackButtonContainer>
-        <BackButton
-          {...(noLink
-            ? {}
-            : {
-                as: Link,
-                to: Urls.dashboard(dashboard),
-              })}
-          round
-          icon="arrow_left"
-          aria-label={label}
-          onClick={handleClick}
-        />
-      </BackButtonContainer>
+    <Tooltip label={label}>
+      <ActionIcon
+        className={DashboardBackButtonS.DashboardBackButton}
+        variant="outline"
+        radius="xl"
+        size="2.625rem"
```

Comment:
`2.625rem` seems a little bit arbitrary, where does this value come from?

Reply:
this comes from the fact that the current dashboard back button is 42px, and I converted it to `rem`. We don't have that value anywhere I think